packages/flutter_test/lib/src/accessibility.dart

```diff
@@ -329,7 +329,9 @@ class MinimumTextContrastGuideline extends AccessibilityGuideline {
           // the last transform layer.
           final double ratio = 1 / renderView.flutterView.devicePixelRatio;
           image = await layer.toImage(renderView.paintBounds, pixelRatio: ratio);
-          return image.toByteData();
+          final Future<ByteData?> data = image.toByteData();
```

Comment:
Can you await here so that we don't dispose before toByteData future resolved?

Reply:
Good catch. Thank you!